controllers/ccruntime_controller.go

```diff
@@ -719,7 +745,7 @@ func (r *CcRuntimeReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 func (r *CcRuntimeReconciler) deleteUninstallDaemonsets() (ctrl.Result, error) {
```

Comment:
note: in a separate PR, it might be worth looking at renaming this method since it is not restricted to Uninstall DS :-)

Reply:
yes, true. I'll open an issue for it